vta/microsites
# Cost Center & Project Number #29

|#138 https://projects.vta.org/projects/copy-center/work_packages/138
|#237 https://projects.vta.org/projects/copy-center/work_packages/237

This PR adds Order Center Number & Project Number to the checkout page. The Cost Center Number provides a searchable dropdown for the user to search and select from. The Project Number input is a text input with no specific format yet since user input is variable. The Placing an order requires filling out one of the two, otherwise, order submission will be rejecting.

Also, Additional styling was added to the **Checkout** page for more clarity.

**Testing:**
Testing will require multiple order submissions.
1. Fill out an order and continue to the **Checkout** page.
2. Search for Cost Center Numbers with numbers, names, or both. You can exclude the dash in the search.
2. Attempt to submit an order without filling out Cost Center Number or Project Number.
3. Then try with **Cost Center Number** OR **Project Number**. If the order is successful, check your order on the admin page and see if the completed field is saved as metadata under **Custom Fields**.

**Requirements:**

### Cost Center
- [x] This input will be placed on the **Checkout** page.
- [x] This input should be a hybrid text input and dropdown menu.
- [x] The Cost Center Number should be searchable.
- [x] Users should not be able to continue with an invalid cost center number.
- [x] If this field is filled out, **Project Number** does not have to be filled out.

### Project Number
- [x] Provide this field as a text input at **Checkout**
- [x] If this field is filled out, the **Cost Center Number** does not have to be filled out
- [x] If neither of the above fields is filled out, then there should be error messages indicated that one of the two fields should be filled out.

### Checkout Styling
- [x] Added an additional "Place Order" button at the top of the page (requested from Copy Center Team)
- [x] Added a container border to make it easier for customers to review order details.
- [x] Added borders between order items to separate items from each other.
